content/en/docs/reference/using-api/cel.md

```diff
@@ -61,13 +61,38 @@ Example CEL expressions:
 | `self.names.size() == self.details.size() && self.names.all(n, n in self.details)` | Validate the 'details' map is keyed by the items in the 'names' listSet           |
 {{< /table >}}
 
-## CEL community libraries
-
-Kubernetes CEL expressions have access to the following CEL community libraries:
-
-- CEL standard functions, defined in the [list of standard definitions](https://github.com/google/cel-spec/blob/master/doc/langdef.md#list-of-standard-definitions)
-- CEL standard [macros](https://github.com/google/cel-spec/blob/v0.7.0/doc/langdef.md#macros)
-- CEL [extended string function library](https://pkg.go.dev/github.com/google/cel-go/ext#Strings)
+## CEL options, language features, and libraries
+
+CEL is configured with the following options, libraries and language features, introduced at the specified Kubernetes versions:
+
+| CEL option, library or language feature             | Included                                                                                                        | Availablity                |
```

Comment:
Do we wanna put the dark mode func also into documentation? It would be easier for people whoever introduce new func instead of remembering to come back and add doc one release later

Reply:
This was discussed previously and the decision was to only add functions that users have access to.